docs/administration/security/project-acl.md

```diff
@@ -0,0 +1,39 @@
+# 3.4.0 ACL Changes
```

Comment:
Overall this is organized as if it is release notes.  We need to integrate this content into the existing content.  These two topics are separate bits of information that likely need to replace or enhance other existing sections.